pkg/scanners/trivy/trivy.go

```diff
@@ -300,5 +300,6 @@ func scan(s Scanner, allImages []unversioned.Image) ([]unversioned.Image, []unve
 		}
 	}
 
+	log.Info("Total number of vulnerabilities found", "Vuln Count", len(vulnerableImages))
```

Comment:
should we add here https://github.com/Azure/eraser/pull/758/files#diff-fe466f8d26e73fa8264bc718c5596bbbb1ac1fe3e124c94b614014575782bb9dR166 (line 166)?